packages/perseus/src/__tests__/mock-asset-loading-widget.tsx

```diff
@@ -8,15 +8,15 @@ import type {WidgetExports} from "../types";
 
 export const mockedAssetItem: PerseusItem = {
     question: {
```

Comment:
This one was a little tricky as it was faking a widget that doesn't exist. I've changed the widget to use the ExampleWidget, as it is one of the simplest widgets, and the test passes without issues.